index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Unit 2 Assessment</title>
+  <script src="./index.js" type="text/javascript"></script>
+  <link rel="stylesheet" href="index.css">
+</head>
+<body>
+  <div class="header">
+  
+  <img class="img" src="/Users/bsihame/Desktop/core/unit2/unit_2_assessment/unit_2_assessment/logo.png">
+  <h1>Ghibli Review App</h1>
+
+</div>
+
+<select name="" id="droplist"></select>
+
+<div>
+<h3 id="titleMovie"></h3>
+<P id="year"></P>
+<p id="description"></p>
+<form action="">
+  <input id="reviewText" type="text">
+  <input id="submit" type="text" value="Submit Review">
+  <ul>
+    <li></li>
+  </ul>
+</form>
+  <div></div>
+</div>
```

Comment:
Good practices you should have followed here are:
  - Write html tags all lowercase
  - Indent the html tags that are inside other html tags. (inner elements)